src/stavenote.ts

```diff
@@ -1086,9 +1162,11 @@ export class StaveNote extends StemmableNote {
       this.stem.adjustHeightForFlag();
     }
 
-    ctx.openGroup('stem', undefined, { pointerBBox: true });
-    this.stem?.setContext(ctx).draw();
-    ctx.closeGroup();
+    if (this.stem) {
```

Comment:
This looks like a separate issue, right?

Reply:
This is to avoid to open the group is there is no stem.